code/WorkInProgress/MechanicMadness.dm

```diff
@@ -344,6 +344,8 @@
 	w_class = 1.0
 	level = 2
 	var/cabinet_banned = false // whether or not this component is prevented from being anchored in cabinets
+	var/one_per_tile = false // if true makes it so that only one component can be wrenched on the tile
+	var/own_id = "/obj/item/mechanics" // im bad at BYOND code okay? (used for one_per_tile, no need to use otherwise)
```

Comment:
```suggestion
```
use .type instead

Reply:
I knew there had to be a way to do it, but i wasnt able to figure it out :)